js/ui/tab_cookies.js

```diff
@@ -156,6 +156,52 @@
           }
         });
         break;
+      case "whitelist-cookie-domain":
+        var domain = getParentData(element, "data-access");
+        var url = getUrl(domain, "", true);
```

Comment:
Question: You get/generate here URL, but it's not being used, same applies below.

In general I'm not sure why you need to determine the actual URL:
- Currently PM is doesn't differentiate between http and https when it comes to domain.

I think if you wish to whitelist domain, you need only domain rather than differentiate between protocols.
Protocols consideration might be a separate issue, if we decide to differentiate between them.

With above said, I think you should be able to achieve your result without implementing `getUrl ` method for this usecase.

Reply:
yup, was not really needed, so i removed it

Reply:
I still can see it, maybe you forgot to push your local changes yet?